package-res/resources/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -89,16 +89,21 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
           var parameterValue = this.dashboard.getParameterValue(this.parameter),
               value = undefined;
 
-          if(parameterValue.length == 1) {
+          // If no value was specified use current date
+          if (!parameterValue.length){
```

Comment:
Why wouldn't you just use  [[ if (!parameterValue) { ]]. Is or can parameterValue be null?